rclpy/CMakeLists.txt

```
@@ -175,7 +175,7 @@ if(BUILD_TESTING)
       test/test_messages.py
       test/test_node.py
       test/test_parameter.py
-      test/test_parameter_client.py
+      test/test_parameter_service.py
```

you don't need to remove that

```suggestion
      test/test_parameter_client.py
      test/test_parameter_service.py
```

---

@ivanpauno Friendly ping. Please look at above my comments.

---

it should be added, right? i think we can keep this file as well.

---

Okay. Keep it

---

yes, we can keep it